### DISCUSSION TITLE
Google Ads UTM Parameters

### DISCUSSION THREAD

**Discussion by rossdav70** [plausible/analytics#3106] [Anything Else]
Hi all,

I'm used to working with Ads and Analytics and being able to clearly see the source of my traffic as paid / organic etc. With plausible it appears my PPC traffic is being tagged as Google and I see noting in the UTM parameters. My understanding was that Google automatically appends destination URLS with UTM parameters if Auto tagging is on (which it is) so how do I get to see this traffic as from Google and the relevant parameters? 

Thanks!

**Comment by metmarkosaric** [comment]
hi @rossdav70! i think when using auto-tagging in Google ads, they're not using UTM parameters but they use their own GCLID parameters. if you try to manually tag your paid ads with UTM parameters you'll see them in your Top Sources report in Plausible and they'll be segregated from the organic traffic. parameters ref, source, utm_source, utm_medium, utm_campaign, utm_content and utm_term are all valid and supported by Plausible out of the box. more here https://plausible.io/docs/manual-link-tagging

**Reply by rossdav70** [reply]
This is helpful @metmarkosaric . Thank you.  I have added a manual tracking template with the tags I want which in theory means I will get the custom UTM tags and the GCLID.

I am out of Ads credit for today so will find out tomorrow Fingers crossed!
